client/src/utils/queries.js

```diff
@@ -2,11 +2,35 @@ import { gql } from "@apollo/client";
 
 export const GET_MOVIES = gql`
   query {
-    movies {
-      title
-      year
+    me {
+      _id
+      username
+      email
+      savedMovies {
+        movieId
+        title
+        year
+        image
+      }
```

Comment:
how is this different from get_me, why are we getting 'me' when this says get movies? get me, should get me. get movies, should get movies.